spyder/plugins/variableexplorer/widgets/collectionseditor.py

```diff
@@ -1438,23 +1437,12 @@ def __init__(self, parent, data, minmax=False, shellwidget=None,
     def get_value(self, name):
         """Get the value of a variable"""
         value = self.shellwidget.get_value(name)
-        # Reset temporal variable where value is saved to
-        # save memory
-        self.shellwidget._kernel_value = None

```

Comment:
I think this is left over from a previous implementation as I couldn't find `_kernel_value`

Reply:
Yes, it was.